feat: add high-scores exercise (#34)
exercism/sqlite: CONTRIBUTING.md

    ```sh
    bin/fetch-configlet
    slug=bob
    bin/configlet create --practice-exercise $slug
    ```

2. create the data.csv file from canonical data

    ```sh

>     cd exercises/practice/$slug

Reviewer: ```suggestion
    cd "exercises/practice/$slug"
```

>     cp "${XDG_CACHE_DIR:-$HOME/.cache}"/exercism/configlet/problem-specifications/exercises/$slug/canonical-data.json .

Reviewer: ```suggestion
    cp "${XDG_CACHE_DIR:-$HOME/.cache}/exercism/configlet/problem-specifications/exercises/$slug/canonical-data.json" .
```

        test_code TEXT,
        task_id INTEGER DEFAULT NULL,
        -- Here are columns for the actual tests
        property TEST NOT NULL,
        input INT NOT NULL,
        expected TEXT NOT NULL
    );

>     -- Note: the strings below _may_ literal tab, newline, carriage returns.

Reviewer: ```suggestion
    -- Note: the strings below _may_ contain literal tabs, newlines, or carriage returns.
```
